src/server/plugins/engine/views/common/warn-missing-notification-email.html

```diff
@@ -0,0 +1,8 @@
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% macro warnMissingNotificationEmail(designerUrl, slug) %}
+    {{ govukWarningText({
+        html: 'If you want to receive a test email, <a target="_blank" class="govuk-link" href=' + designerUrl + '/library/' + slug + '/edit/notification-email>enter the email address (opens in new tab)</a> you want form submissions to go to.',
```

Comment:
Can you use a separate `{% set html %}` for the `html` property like in [**preview-banner.html**](https://github.com/DEFRA/forms-runner/blob/main/src/server/plugins/engine/views/partials/preview-banner.html)?

Makes it a bit easier/safer than string joining HTML

Reply:
done